packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.controller.js

```diff
@@ -108,17 +117,34 @@ export default class PciServingNamespaceModelsAddController {
     this.error = false;
     this.isAdding = true;
 
-    this.OvhManagerPciServingModelsService.add(
-      this.projectId,
-      this.namespaceId,
-      {
+    let modelCreation;
+    if (this.model.workflowTemplate === this.PRESET_IMAGE) {
+      // Preset image
+      modelCreation = {
         id: this.model.id,
         storagePath: this.model.storagePath,
         flavor: this.model.flavor.id,
         workflowTemplate: this.model.workflowTemplate,
-        imageId: get(this.model.imageId, 'id'),
+        imageId: this.model.image.id,
```

Comment:
**Note**: Contrary to before, if `this.model.image` is undefined this will crash